### PR TITLE
MCR-2415 extract rules from mcr access interface

### PR DESCRIPTION
**Pull request by sebhofmann**
[Link to jira](https://mycore.atlassian.net/browse/MCR-2415).

### CODE REVIEW

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on ce9f3a9a1c1579c523a7de56923d9512646012bb! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/7180-pull_request-1252

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on 8b678e0a478731efd8a76e5d793c7a7ee09d787d! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/7182-pull_request-1252

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on 39da8b33f59bb27174710af22b9ae252e6563170! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/7184-pull_request-1252

File: mycore-base/src/main/java/org/mycore/access/MCRAccessManager.java [hunk 1]
```diff
@@ -388,7 +388,11 @@ public static Element getFalseRule() {
      *            the access permission for the rule
      */
     public static boolean hasRule(String id, String permission) {
-        return getAccessImpl().hasRule(id, permission);
+        if(getAccessImpl() instanceof MCRRuleAccessInterface) {
```
Comment (kkrebs) [inline mycore-base/src/main/java/org/mycore/access/MCRAccessManager.java:392]: can you add some javadoc here

**Review by kkrebs — APPROVED** [review]

**Pull request merged**